### ISSUE TITLE
Conventional lookups based on log-derivative

### ISSUE THREAD

**Issue by ledwards2225** [AztecProtocol/barretenberg#959]
Tldr: Without structured trace? optional. With structured trace? Necessary

Committing to the lookup grand product is one of our highest commitment costs (equivalent to permutation grand product) so converting to log-deriv would be beneficial even without the structured trace optimization. With a structured trace, however, it becomes an unacceptably large cost because it is not constant over "dead" ranges, i.e. it is dense and nontrivial across the full size of the trace. This is in contrast to the permutation grand product which takes constant values in the dead regions and can thus be committed to relatively cheaply via the sorted MSM optimization.

**Comment by ledwards2225** [comment]
Closed with this [PR](https://github.com/AztecProtocol/aztec-packages/pull/7020)
